parts/linux/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -302,4 +302,14 @@ systemctlDisableAndStop() {
         systemctl_disable 20 5 25 $1 || echo "$1 could not be disabled"
     fi
 }
+
+# return true if a >= b 
+semverCompare() {
+    VERSION_A=$1
+    VERSION_B=$2
+    [[ "${VERSION_A}" == "${VERSION_B}" ]] && return 0
+    sorted=( $( echo ${VERSION_A} ${VERSION_B} | tr ' ' '\n' | sort -V ) )
```

Comment:
I think we should trim the versions before the equality comparison?

Reply:
yea im assuming the version strings passed in are already trimmed. i can add a trim in here as well to be safe